frontend/src/app/reducers/varianttests.js

```diff
@@ -2,6 +2,10 @@
 
```

Comment:
All that changed in this file is instead of the test-choosing happening synchronously at the top level of the module, it happens in a chooseTests function that is triggered by an action once we are running in the browser.